pkgs/development/python-modules/setuptools-scm/tests.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, setuptools-scm
+, pytestCheckHook
+, git
+, mercurial
+, pip
+, virtualenv
+}:
+
+buildPythonPackage rec {
```

Comment:
Would it not be better to do a `.overrideAttrs` here? If someone adds any e.g. patches or other quirks to the parent package, that won't get reflected in the test run.

Reply:
We are actually testing the package passed into `checkInputs` here, so patches will get reflected.